app/charts/map/wmts-utils.ts

```diff
@@ -154,7 +154,8 @@ export const getWMTSTile = ({
     id: `tile-layer-${customLayer.url}`,
     beforeId,
     data: getWMTSLayerData(customLayer.url, {
-      identifier: wmtsLayer.dimensionIdentifier,
+      // TODO, Understand the implications of the empty string here
+      identifier: wmtsLayer.dimensionIdentifier ?? "",
```

Comment:
In the original implementation, we needed identifier to dynamically replace the temporal value (connection with time slider), see:

```ts
const getWMTSLayerData = (
  url: string,
  { identifier, value }: { identifier: string; value: string | number }
) => {
  return url
    .replace(`{${identifier}}`, `${value}`)
    .replace("{TileMatrix}", "{z}")
    .replace("{TileCol}", "{x}")
    .replace("{TileRow}", "{y}");
};
```

I think it's a bit weird for it to be empty, as from what I understood that's the whole point of WMTS layers to have this temporal value embedded inside the URL (but maybe in some cases they are just broken, or I missed something 😅).

Reply:
I removed the replacement of identifier if identifier is passed as undefined.